### DISCUSSION TITLE
Is it possible to modify settings on the Tailwind CSS extension?

### DISCUSSION THREAD

**Discussion by estenp** [zed-industries/zed#19317] [Feature Requests]
How does Tailwind support work in Zed?

In VSCode there is an extension that provides autocomplete/intellisense for TW classes.

Zed seems to provide basically the same extension, but it's not a Zed extension.

The VSCode extension can be modified through the `.vscode/settings.json` file to change the default font size (which affects a comment revealing the pixel calculation on rem values:

```
{ "tailwindCSS.rootFontSize": 12 }
```

This results in the TW intellisense accurately calculating rem values based on the provided root font size:
![image](https://github.com/user-attachments/assets/c5844c51-4ad3-4f4f-928a-315f8033a388)

How can we configure the root font size for Tailwind intellisense in Zed?
![image](https://github.com/user-attachments/assets/5f252078-0d53-471f-8a0e-f52832b5c030)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
